src/sentry/static/sentry/app/views/settings/components/forms/jsonForm.tsx

```diff
@@ -128,12 +129,59 @@ class JsonForm extends React.Component<Props, State> {
     }
   }
 
+  shouldDisplayForm = (fields: FieldObject[]) => {
```

Comment:
This doesn't need to be a bound arrow function right?

Reply:
yes, but please do you see any downsides in using it?

Reply:
Yes it allocates the function on every instance of the class, instead of just once for the class prototype.

Reply:
oh interesting...I didn't know about it